terraform/docker/python_requirements

```diff
@@ -1,5 +1,5 @@
 #  
-boto3~=1.26
+boto3~=1.34
 requests~=2.27
```

Comment:
2.32

Reply:
and:
pipenv~=2023.4 => 2024.0
python-hcl2~=2.0 ## TODO 4.3.0 https://github.com/amplify-education/python-hcl2
pre-commit~=3.3.1 ?=> 3.8.0
azure-cli~=2.48 ==> 2.63

Reply:
🤦